app/controllers/weets.js

```diff
@@ -0,0 +1,6 @@
+const jokes = require('../services/jokes_api');
+
+exports.weets = async (req, res) => {
```

Comment:
You could name this controller file for example `jokes.js`, it is more representative with the functionality.
We don't return weets, we are just returning jokes. Do you understand me?